test/e2e/apps/daemon_restart.go

```diff
@@ -298,8 +299,12 @@ var _ = SIGDescribe("DaemonRestart [Disruptive]", func() {
 
 	ginkgo.It("Kubelet should not restart containers across restart", func() {
 
-		nodeIPs, err := framework.GetNodePublicIps(f.ClientSet)
-		framework.ExpectNoError(err)
+		nodeIPs, err := e2enode.GetPublicIps(f.ClientSet)
+		if err != nil {
+			e2elog.Logf("Unexpected error occurred: %v", err)
+		}
+		// TODO: write a wrapper for ExpectNoErrorWithOffset()
```

Comment:
Sure, will do

Reply:
Keeping `OrDie` for `GetReadySchedulableNodesOrDie` because we are still using `framework.GetReadySchedulableNodesOrDie()` in the tests, and change to `e2enode` will make this PR a XXL size.. So I just copy this method here and I plan to do in a following PR to fix the references of this method.

Reply:
ah sorry, you did mentioned that at the beginning of the PR.